src/screens/WebScreen.js

```diff
@@ -1,21 +1,23 @@
 import { noop } from 'lodash';
 import PropTypes from 'prop-types';
 import React, { useContext, useEffect } from 'react';
-import { ActivityIndicator } from 'react-native';
+import { ActivityIndicator, StyleSheet, TouchableOpacity } from 'react-native';
 import { WebView } from 'react-native-webview';
 
 import appJson from '../../app.json';
 import { LoadingContainer, SafeAreaViewFlex } from '../components';
-import { colors, consts } from '../config';
+import { colors, consts, Icon, normalize } from '../config';
+import { onDownloadAndSharePdf } from '../helpers';
 import { useTrackScreenViewAsync } from '../hooks';
 import { NetworkContext } from '../NetworkProvider';
 
 const { MATOMO_TRACKING } = consts;
 
-export const WebScreen = ({ route }) => {
+export const WebScreen = ({ navigation, route }) => {
   const { isConnected } = useContext(NetworkContext);
   const trackScreenViewAsync = useTrackScreenViewAsync();
   const webUrl = route.params?.webUrl ?? '';
+  const documentData = route.params?.documentData ?? '';
```

Comment:
my test pdf cannot be shown, just white screen:

<img width="279" alt="image" src="https://github.com/user-attachments/assets/af120554-6e80-44d5-8d50-35bb4321c6b9">

Reply:
Unfortunately this problem is not caused by our code but by the google masking we use to display it in `WebScreen`. I have tried many times and I don't understand why, sometimes the PDF appears and sometimes it doesn't. I have the same problem with the browser on my computer. So I integrated the `react-native-pdf` package and created `PdfScreen`.